README.md

```diff
@@ -40,13 +40,11 @@ sources, and line numbers of addresses involved in a process.
 
   let process_id: u32 = std::process::id(); // <some process id>
   // load all symbols of loaded files of the given process.
-  let sym_srcs = [
-    SymbolSrcCfg::Process(cfg::Process { pid: Some(process_id) })
-  ];
+  let cfg = SymbolSrcCfg::Process(cfg::Process { pid: Some(process_id) });
```

Comment:
nit (and feel free to disagree): `pid: Some(process_id)` seems unnecessarily verbose, to be honest, I find `pid: process_id` and `pid: 0` (for self) just as obvious, but more succinct.

Reply:
I think it's reasonable to make it explicit that we don't *need* a PID and this is the straight forward way to express that in Rust. So I'd prefer we leave it.

Reply:
my unease with `pid: None` is due to the fact that we *always* need PID, it's just we have a special "self" short-cut. It is still a PID. `pid: None` reads like "Process context with no PID", which isn't true, of course. But it's my interpretation. I'm fine leaving it as is.

Reply:
I will have changes in the works that make the existing `Pid` type a first class citizen in our APIs. Then we no longer need an `Option`. Should be out soon.

Reply:
Included now here: https://github.com/libbpf/blazesym/pull/134
Let me know if those make sense.